src/ExpressionEvaluator/CSharp/Source/ResultProvider/CSharpResultProvider.csproj

```diff
@@ -36,11 +36,6 @@
     <Compile Include="CSharpFormatter.Values.cs" />
```

Comment:
Consider adding `<!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->` at the top of the file after the xml declaration.